cb-talent-development/employer-style-base
# EM-1315: Small Business Page and Header

**Purpose:**
> As a user looking to buy perpetual product, I want to come to a page where I can purchase product, compare plans and learn more about the benefits of CareerBuilder subscription products.

* Refactos existing product header mixin
* Adds two text style options `--left` and `--centered`

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/EM-1315

**Changes:**
* Changes to setup, Architectural changes, Migrations,Side effects, Library changes
  * n/a

**Screenshots**
n/a


**QA Links:**
http://web.employer-4.development.c66.me/recruiting-solutions/small-business-subscription-plans-and-pricing

**How to Verify These Changes**
* Specific pages to visit
  * http://web.employer-4.development.c66.me/recruiting-solutions/small-business-subscription-plans-and-pricing

Affected by refactoring:
  * http://web.employer-4.development.c66.me
  * http://web.employer-4.development.c66.me/recruiting-solutions/job-postings
  * http://web.employer-4.development.c66.me/recruiting-solutions/resume-database

* Steps to take
  * Verify headers on homepage, Jobs and Search pages didn't change
  * Compare Small Business header to JIRA story and mockups
     * Check various browsers and devices

* Responsive considerations
  * see that header resizes responsively

**Relevant PRs/Dependencies:**
https://github.com/cbdr/employer/pull/837

**Additional Information**
